SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/Storage.cs

```diff
@@ -117,12 +122,7 @@ public Task DeleteLogs(string channelName, string batchId)
                         _pendingDbIdentifiers.Remove(id);
                     }
                     AppCenterLog.Debug(AppCenterLog.LogTag, deletedIdsMessage);
-                    foreach (var id in identifiers)
-                    {
-                        _storageAdapter
-                            .DeleteAsync<LogEntry>(entry => entry.Channel == channelName && entry.Id == id)
-                            .GetAwaiter().GetResult();
-                    }
+                    _storageAdapter.DeleteAsync(TableName, $"{ColumnChannelName} = \'{channelName}\' AND {ColumnIdName} IN ({string.Join(",", identifiers)})").GetAwaiter().GetResult();
```

Comment:
We are not allowed to craft dynamic queries when we can use prepared statements.

Reply:
Can you elaborate please? I can't really see the dynamic query here, just the dynamic `whereClause` constructed here, and I can't think of any ways to re-using our existing code without using it.

Reply:
@AnastasiaKubova @Jamminroot as discussed offline, please use bind_* API to set values